### PR TITLE
add StreetAddress Migration plus sonar fix IgnoreAntiForgeryToken

### PR DESCRIPTION
**Pull request by DanielHodgeGH**

### CODE REVIEW

File: src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditStreetAddress.razor [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      Required="true"
+                      RequiredError="@ErrorRequiredMessage(TitlePostalCode)"
+                      TextChanged="OnPostalCodeChanged"
+                      MaxLength="@MaxLengthPostalCode"
+                      Value="CurrentPostalCode" />
+    </MudItem>
+
+    @if (CountrySelectionList != null && CountrySelectionList.Count > 0)
```
Comment (anna-naumova25) [inline src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditStreetAddress.razor]: Can we make CountrySelectionList non nullable list, empty by default, so we can just call CountrySelectionList .Any()

**Review by anna-naumova25 — DISMISSED** [review]
All good, just one comment

File: src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditStreetAddress.razor (hunk 1 above)
Comment (DanielHodgeGH) [reply src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditStreetAddress.razor]: on latest commit

File: src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewStreetAddress.razor.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+public partial class ViewStreetAddress : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public StreetAddressModel? StreetAddress { get; set; }
+
+    [Parameter]
+    public List<CountryModel>? CountrySelectionList { get; set; }
```
Comment (anna-naumova25) [inline src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewStreetAddress.razor.cs:16]: sorry @DanielHodgeGH  could you update this one as well? I think we have a nox convention for List to be empty by default (not null). Do it in the next PR if you wish.

**Review by anna-naumova25 — APPROVED** [review]

**Review by rochar — APPROVED** [review]

File: src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewStreetAddress.razor.cs (hunk 2 above)
Comment (DanielHodgeGH) [reply src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewStreetAddress.razor.cs:16]: yeah no problem i'll change this on the next PR thanks

**Pull request merged**